### PR TITLE
preparations to rename master branch to main

### PR DESCRIPTION
**Pull request by dlockhart**
We'd like to rename BSI's `master` branch to `main`, and these changes are needed before that can happen. There's also the [sync job in the LMS](https://github.com/Brightspace/lms/blob/master/.github/workflows/bsi-sync.yml), but since BSI communicates with it, it doesn't appear to require any updates.

### CODE REVIEW

File: .github/workflows/bsi-update.yml [hunk 1]
```diff
@@ -30,6 +30,5 @@ jobs:
           AUTO_MERGE_TOKEN: ${{ secrets.BSI_AUTO_APPROVE_GITHUB_TOKEN }}
           BRANCH_NAME: ghworkflow/bsi_auto_update
           COMMIT_MESSAGE: 'BSI Auto Update Dependencies'
-          DEFAULT_BRANCH: master
```
Comment (dlockhart) [inline .github/workflows/bsi-update.yml:33]: `main` is the default value for this parameter

**Review by mdulat — APPROVED** [review]
look good!

**Comment by dlockhart** [comment]
Interesting CI isn't going to run in Travis because of the branch name change haha

**Review by svanherk — APPROVED** [review]

**Comment by svanherk** [comment]
> Interesting CI isn't going to run in Travis because of the branch name change haha

Probably a good thing so a release isn't triggered and all the other stuff runs and breaks?

**Pull request merged**